<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>572</width>
    <height>476</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Max Angle</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="sBox_maxAngleA">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="suffix">
           <string> degree</string>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>270</number>
          </property>
          <property name="value">
           <number>35</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_13">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Min Angle</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="sBox_minAngleA">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="suffix">
           <string> degree</string>
          </property>
          <property name="maximum">
           <number>269</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_iniA">
          <property name="text">
           <string>Initial angle:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="sBox_initialAngleA">
          <property name="suffix">
           <string> degree</string>
          </property>
          <property name="maximum">
           <number>180</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_6">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Servo No.</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="sBox_addrA">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="prefix">
           <string>No. </string>
          </property>
          <property name="maximum">
           <number>11</number>
          </property>
          <property name="value">
           <number>6</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Motor A :</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDial" name="dial_motorA">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="maximum">
           <number>180</number>
          </property>
          <property name="value">
           <number>90</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label_7">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Max Angle</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="sBox_maxAngleB">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="suffix">
           <string> degree</string>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>270</number>
          </property>
          <property name="value">
           <number>180</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_14">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Min Angle</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="sBox_minAngleB">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="suffix">
           <string> degree</string>
          </property>
          <property name="maximum">
           <number>269</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_iniB">
          <property name="text">
           <string>Initial angle:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="sBox_initialAngleB">
          <property name="suffix">
           <string> degree</string>
          </property>
          <property name="maximum">
           <number>180</number>
          </property>
          <property name="value">
           <number>90</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_8">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Servo No.</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="sBox_addrB">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="prefix">
           <string>No. </string>
          </property>
          <property name="maximum">
           <number>11</number>
          </property>
          <property name="value">
           <number>7</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Motor B:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDial" name="dial_motorB">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="maximum">
           <number>180</number>
          </property>
          <property name="value">
           <number>90</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="label_9">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Max Angle</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="sBox_maxAngleC">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="suffix">
           <string> degree</string>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>270</number>
          </property>
          <property name="value">
           <number>180</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_15">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Min Angle</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="sBox_minAngleC">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="suffix">
           <string> degree</string>
          </property>
          <property name="maximum">
           <number>269</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_iniC">
          <property name="text">
           <string>Initial angle:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="sBox_initialAngleC">
          <property name="suffix">
           <string> degree</string>
          </property>
          <property name="maximum">
           <number>180</number>
          </property>
          <property name="value">
           <number>90</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_10">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Servo No.</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="sBox_addrC">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="prefix">
           <string>No. </string>
          </property>
          <property name="maximum">
           <number>11</number>
          </property>
          <property name="value">
           <number>8</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Motor C:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDial" name="dial_motorC">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="maximum">
           <number>180</number>
          </property>
          <property name="value">
           <number>90</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QLabel" name="label_11">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Max Angle</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="sBox_maxAngleD">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="suffix">
           <string> degree</string>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>270</number>
          </property>
          <property name="value">
           <number>180</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_16">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Min Angle</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="sBox_minAngleD">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="suffix">
           <string> degree</string>
          </property>
          <property name="maximum">
           <number>269</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_iniD">
          <property name="text">
           <string>Initial angle:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="sBox_initialAngleD">
          <property name="suffix">
           <string> degree</string>
          </property>
          <property name="maximum">
           <number>180</number>
          </property>
          <property name="value">
           <number>90</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_12">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Servo No.</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="sBox_addrD">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="prefix">
           <string>No. </string>
          </property>
          <property name="maximum">
           <number>11</number>
          </property>
          <property name="value">
           <number>9</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Motor D:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDial" name="dial_motorD">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="maximum">
           <number>180</number>
          </property>
          <property name="value">
           <number>90</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="pBtn_begin">
        <property name="text">
         <string>Begin</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pBtn_config">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Config</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>572</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
